sysutils/pfSense-pkg-apcupsd/Makefile

```diff
@@ -0,0 +1,49 @@
+# $FreeBSD$
+
+PORTNAME=	pfSense-pkg-apcupsd
+PORTVERSION=	0.3.9
+CATEGORIES=	sysutils
+MASTER_SITES=	# empty
+DISTFILES=	# empty
+EXTRACT_ONLY=	# empty
+
+MAINTAINER=	coreteam@pfsense.org
+COMMENT=	pfSense package apcupsd
+
+LICENSE=	ESF
+
+RUN_DEPENDS=	${LOCALBASE}/sbin/apcupsd:sysutils/apcupsd \
+		/etc/inc/phpmailer/PHPMailerAutoload.php:mail/pfSense-pkg-mailreport
```

Comment:
Well, it's not. It will send just text.

I could change it to send_stmp_message() 
https://github.com/pfsense/pfsense/blob/master/src/etc/inc/notices.inc#L309

What do you think ?